src/script/view_model/list/StartUIViewModel.ts

```diff
@@ -132,6 +133,7 @@ export class StartUIViewModel {
     this.peopleTabActive = ko.pureComputed(() => this.state() === StartUIViewModel.STATE.ADD_PEOPLE);
 
     this.submittedSearch = false;
+    this.federationDomain = Config.getConfig().FEATURE.FEDERATION_DOMAIN;
```

Comment:
Is this the domain or the name of the domain?

Reply:
The domain, like `wire.com`

Reply:
Ah, because it the ticket they want something like a domain name I believe. Maybe that's something that has been discussed internally?

Reply:
Resolved 1:1